src/tests/variorum/CMakeLists.txt

```diff
@@ -6,16 +6,16 @@
 set(BASIC_TESTS
     t_variorum_cap_best_effort_node_power_limit
     t_variorum_cap_gpu_power_ratio
-    t_variorum_cap_socket_frequency
+    t_variorum_cap_socket_frequency_limit
     t_variorum_cap_socket_power_limit
     t_variorum_monitoring
     t_variorum_poll_data
-    t_variorum_query_clock_speed
+    t_variorum_query_frequency
```

Comment:
I wonder if we should rename this test to be query_frequency_limit? I can't remember if the test prints out the current frequency or the available frequency limits that a user could set.

Reply:
I think it's appropriate to leave the test name as is since depending on the architecture, this may return the frequency limit for the domain or may give the current frequency.